docs/src/jsx/allLiveEditors.js

```diff
@@ -10,7 +10,7 @@ import React from "react";
 
 import WorldviewCodeEditor from "./utils/WorldviewCodeEditor";
 
-function makeCodeComponent(raw, componentName, isRowView) {
+function makeCodeComponent(raw, componentName, { isRowView, shouldInsertCodeSandboxStyleFix } = {}) {
```

Comment:
#nit: Not sure if the "Fix" part of this variable is necessary or describing what it is doing. Do you think `insertCodeSandboxStyle` would be sufficient / more descriptive?

Reply:
yeah, totally :) Updated.